chapters/sampling-randomization-power.tex

```diff
@@ -398,8 +450,77 @@ \subsection{Minimum sample size}
 than an output for reporting requirements.
```

Comment:
This sentence seems garbled, like maybe a line is missing "is it better to add another cluster, than an output for reporting requirements"

Reply:
?